os/manual-rollbacks.md

```diff
@@ -2,6 +2,8 @@
 
 As much as we all love flawless, automatic updates, there may be occasions when an update must be rolled back. This is fairly straightforward on Container Linux, once you know the right commands.
 
+**Note**: Container Linux allows you to rollback to a previously active partition, or change to an older release channel (to downgrade to a stable version). It does not allow you to downgrade to an uninstalled, user defined version number.
```

Comment:
Immediately after an upgrade reboot, the active version of Container Linux can be rolled back to the version installed on the rollback partition, or downgraded to the version current on any lower release channel. Rolling back to an arbitrary previous version number is not supported.